receiver/postgresqlreceiver/integration_test.go

```diff
@@ -1,9 +1,6 @@
 // Copyright The OpenTelemetry Authors
 // SPDX-License-Identifier: Apache-2.0
 
-//go:build integration
-// +build integration
-
```

Comment:
We need a better way to run tests locally without removing it :)

Reply:
I'll follow up https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/22808 by proposing to rename the make target. I never remember the name offhand.

Reply:
<img width="1119" alt="Screenshot 2023-06-02 at 10 05 27 AM" src="https://github.com/open-telemetry/opentelemetry-collector-contrib/assets/16758/b600dd23-4d14-4278-b02c-82517dfdf417">
I end up checking that box in my IDE, though it looks like you run this test from CLI? I hope it helps.